src/main/java/seedu/address/logic/commands/DeleteModuleCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.messages.ModuleMessages.MESSAGE_DELETE_MODULE_SUCCESS;
```

Comment:
Very very small nit, but usually it is not convention to import static variables/methods unless frequently used. See `use it when you require frequent access to static members from one or two classes.` (reference: https://docs.oracle.com/javase/8/docs/technotes/guides/language/static-import.html).

Just extra knowledge hehe :) no need to change unless you want to